Lesson_18 Requests

eBelyakovich opened the pull request:
Рассмотреть gunicorn (wsgi сервер) требования для wsgi стандарта. 
![изображение_2024-07-24_135748149](https://github.com/user-attachments/assets/488bc475-262e-418d-8f7f-11768288b943)
Попробовать реализовать простейшее серверное приложение. 
Приложение получает число с формы и возращает его квадрат

eBelyakovich committed:
Lesson_18 Requests
Рассмотреть gunicorn (wsgi сервер) требования для wsgi стандарта.Попробовать реализовать простейшее серверное приложение.
Приложение получает число с формы и возращает его квадрат

eBelyakovich committed:
lesson19_temp

eBelyakovich committed:
cont

eBelyakovich committed:
Merge pull request #4 from bossniger/temp

Temp

eBelyakovich committed:
lesson19 is done

bossniger commented:
Реализовать, с использованием библиотеки Flask или без, сетевой интерфейс (API) который позволил бы управлять приложением БЖУ
Готово!

eBelyakovich committed:
lesson20 (SQLAlchemy) is done

eBelyakovich committed:
Создать проект Blog с тремя приложениями:
Article
Author
Comment
Создать модели в каждом из них, создать миграции и запустить их.
При этом модели должны быть связаны
